### ISSUE TITLE
Gemini rejects tool JSON schema with array of types

### ISSUE THREAD

**Issue by ianb** [google/generative-ai-docs#444]
### Description of the bug:

I have a tool in my prompt that looks like this (abbreviated):

```json
{
    "name": "describeProblem",
    "description": "Describe the problem in simpler terms for future analysis",
    "parameters": {
        "type": "object",
        "required": [
            "correctAnswerFields"
        ],
        "properties": {
            "correctAnswerFields": {
                "type": [
                    "string",
                    "array"
                ],
                "description": "Format it as a list of fields"
            }
        }
    }
}
```

I get an error like this:

```
Invalid JSON payload received. Unknown name \"type\" at 'tools[0].function_declarations[0].parameters.properties[2].value': Proto field is not repeating, cannot start list.", "status": "INVALID_ARGUMENT", "details": [ { "@type": "type.googleapis.com/google.rpc.BadRequest", "fieldViolations": [ { "field": "tools[0].function_declarations[0].parameters.properties[2].value", "description": "Invalid JSON payload received. Unknown name \"type\" at 'tools[0].function_declarations[0].parameters.properties[2].value': Proto field is not repeating, cannot start list." } ] } ] } } ]
```

A JSON schema is allowed to have an array of `type`. Both Gemini Flash and 1.5 behave the same way.
### Actual vs expected behavior:

Actual behavior: returns error

Expected behavior: respects JSON schema

**Comment by singhniraj08** [comment]
@ianb, Thank you reporting this issue.
This repository is for issues related to website(https://ai.google.dev/) like documentation bugs or improvements. For issues related to Gemini, we would suggest you to use "Send Feedback" option in [Gemini docs](https://ai.google.dev/gemini-api/docs/api-key). Ref: Screenshot below. You can also post this issue on [Discourse forum](https://discuss.ai.google.dev/).

![image](https://github.com/google/generative-ai-docs/assets/110809167/7d11fa96-c1dc-45eb-bb7c-ec8d6609c383)
